.github/workflows/tag-and-release.yaml

```
@@ -0,0 +1,28 @@
+        uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm ci
+      - name: Build
+        run: npm run package
+      - name: Test Local Action
+        uses: ./
```

actionlint

file "dist/index.js" does not exist in "/inmem/40/18f7c18b-d5dc-4a5d-862c-738a9ce4888f/home/jailuser/git". it is specified at "main" key in "runs" section in "Release Automation Action" action

(action)